src/include/rbd/librbd.h

```diff
@@ -144,6 +144,8 @@ enum {
   RBD_IMAGE_OPTION_JOURNAL_ORDER = 5,
   RBD_IMAGE_OPTION_JOURNAL_SPLAY_WIDTH = 6,
   RBD_IMAGE_OPTION_JOURNAL_POOL = 7,
+  RBD_IMAGE_OPTION_FEATURES_SET = 8,
```

Comment:
Currently, --image-shared is taking the use of RBD_IMAGE_OPTION_FEATURES_CLEAR. Because it want to clear some bits from inherited features.